neard/src/config.rs

```diff
@@ -932,18 +938,23 @@ pub fn get_genesis_url(chain_id: &String) -> String {
 pub fn download_genesis(url: &String, path: &PathBuf) {
     info!(target: "near", "Downloading config file from: {} ...", url);
 
-    let mut file = File::create(path).expect("Failed to create / write a config file.");
-    let mut easy = Easy::new();
+    let url: String = url.clone();
+    let path: PathBuf = path.clone();
+
+    actix::System::builder().build().block_on(async move {
+        let client = actix_web::client::Client::new();
+
+        // Create request builder, configure request and send
+        let mut response = client.get(url).send().await.unwrap();
+
+        let body = response.body().limit(250_000_000).await.unwrap();
```

Comment:
Let's add a comment here about the Overflow error, so if we ever see that error message we immediately know what the reason is.